app/controllers/pulitzer/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class Pulitzer::ApplicationController < Pulitzer.base_controller
+  helper Pulitzer::Engine.helpers
+  layout Pulitzer.layout
+  before_action :authenticate_user!
+
+private
```

Comment:
hmm, shouldn't this already be handled by Pulitzer.base_controller? In InvitedHome it would be handled already.